src/base/install/removeTempFiles.m

```diff
@@ -0,0 +1,49 @@
+function removeTempFiles(directory, oldcontent, varargin)
+% Removes all files that are in the specified directory but not part of
+% the oldcontent. By default only removes those files which match files
+% mentioned in the COBRA .gitignore file.
+% USAGE:
```

Comment:
There must be an empty line here. May you also output a status?

Reply:
What should a status indicate?

Reply:
For instance, `status` could be `1` if successful, `0` if there was an issue.